src/doc/rustc/src/codegen-options/index.md

```diff
@@ -209,6 +209,19 @@ Note that while the `-C instrument-coverage` option is stable, the profile data
 format produced by the resulting instrumentation may change, and may not work
 with coverage tools other than those built and shipped with the compiler.
 
+## jump-tables
+
+This option is used to allow or prevent the codegen backend from creating
+jump tables when lowering switches.
```

Comment:
Question [GUARANTEE 2/3]: This wording reads like a _guarantee_ -- but _is it_? _Can_ we make that guarantee?

Reply:
It should, though that is entirely dependent on the codegen backend to honor it.

More accurately, this should be reworded as `the LLVM codegen backend`. However, it seems relatively trivial to pass this through to the gcc backend too.

Reply:
I would probably add "the LLVM codegen backend" for now.